html/json/WS.js

```diff
@@ -270,8 +270,26 @@ var WS = {
 		$.each($("[sbDisplay]"), function(idx, elem) {
 			elem = $(elem);
 			var paths = WS.getPaths(elem, "sbDisplay");
-			if (paths.length > 0)
-				WS.Register(paths, { element: elem, modifyFunc: window[elem.attr("sbModify")] });
+			if (paths.length > 0) {
+				// When there's multiple names, use the
+				// first non-empty one.
+				var mf = function(k, v) {
+					var v = null;
```

Comment:
The local variable shouldn't have the same name as the function parameter.

Reply:
Done